pmpsui/preemptive_requests.py

```diff
@@ -189,6 +189,25 @@ def setup_requests(self):
                     # Keep the calculated value under the "Transmission" header
                     widget.embedded_widget.transmission_label.hide()
 
+                # special setup for the veto indicator
+                # can't use a channel since it's an array
+                veto_byte = widget.findChild(
+                    PyDMByteIndicator,
+                    "veto_byte",
+                )
+                veto_channel_name = Template(
+                    'ca://${P}${ARBITER}:AP:Entry:${POOL}:Veto_RBV'
```

Comment:
I'm not convinced that this PV ever has any nonzero data on it.

I'm suspicious because the veto bit for a pre-emptive request is a single boolean value, not an array.

There are definitely things vetoed right now on KFE according to the arbiter outputs tab (and, typically, either TMO, RIX, or both will be vetoed)). I'm going to dig into the PLC source a bit to understand what this is supposed to be.

Reply:
Yeah, these are always zero for arbitration requests. The bits here are typically used for the main arbiter PLC to tell e.g. the motion PLC which veto devices are active.

Arbiters are vetoed all at once on a single bool that isn't traceable from outside the PLC.
Some relevant links:
- https://github.com/pcdshub/lcls-plc-tmo-motion/blob/671cd29d1a746a40ce225f40cd702c94234bcc5e/plc-tmo-motion/tmo_motion/POUs/PRG_3_PMPS_POST.TcPOU#L30
- https://github.com/pcdshub/lcls-twincat-pmps/blob/master/lcls-twincat-pmps/PMPS/MajorComponents/Arbiter/FB_VetoArbiter.TcPOU

Reading back the linked Jira, I have a note there that says that this isn't possible without PLC work. I probably meant that the Arbiter needs to know and report that it has been vetoed, which it currently does not.